mbedtls-sys/build/bindgen.rs

```diff
@@ -116,6 +118,17 @@ impl super::BuildConfig {
             };
         }
 
+        // In order to support being used in crates that utilize custom
+        // targets,
+        // https://docs.rust-embedded.org/embedonomicon/custom-target.html
+        // a target override is provided for bindgen usage.  Bindgen utilizes
+        // libclang and the current `TARGET` to parse the C files.  If the
+        // `TARGET` is custom, the C stdlib headers will not exist resulting
+        // in parsing errors.
```

Comment:
```suggestion
        // Bindgen utilizes libclang and the current `TARGET` to parse the C files.
        // When the `TARGET` is custom, we need to override it so that bindgen
        // finds the right stdlib headers.
        // See https://docs.rust-embedded.org/embedonomicon/custom-target.html
        // for more details.
```